src/main/webapp/sass/_core.scss

```diff
@@ -3657,4 +3657,9 @@ div.border {
 		justify-content: end;
 		text-align: end;
 	}
+}
+
+// override SynapseTable style when embedded in a wiki
+#body .markdown .SynapseTable {
+	min-height: unset;
```

Comment:
Has a min-height from SRC, let's override it if the SynapseTable is embedded in a wiki.